src/plugins/chart_expressions/expression_metric/public/components/metric_component.tsx

```diff
@@ -116,7 +115,7 @@ class MetricVisComponent extends Component<MetricVisComponentProps> {
     const max = (last(config.colorsRange) as Range).to;
     const colors = this.getColors();
     const labels = this.getLabels();
-    const metrics: MetricVisMetric[] = [];
```

Comment:
nit: Can we keep the old name? I know it's a bit long but just `Metric` will be hard to use outside of the plugin

Reply:
> > Could you also add the vis editors team as codeowners?
> 
> https://github.com/elastic/kibana/blob/f38230fe5241104e97bf1afe1f1462fe829d2161/src/plugins/chart_expressions/expression_metric/kibana.json#L7
> 
> Isn't in enough? If not, could you please help me with it? Thanks)

Thanks, done.

Reply:
We had also done in for the tagcloud expression if you remember :) 
This is the file that you should update https://github.com/elastic/kibana/blob/master/.github/CODEOWNERS

Reply:
> nit: Can we keep the old name? I know it's a bit long but just `Metric` will be hard to use outside of the plugin

Exported as `MetricVisMetric` from the `common` folder.

Reply:
As this is a dimension, can we call it `MetricDimension`?

Reply:
https://github.com/elastic/kibana/blob/f38230fe5241104e97bf1afe1f1462fe829d2161/src/plugins/chart_expressions/expression_metric/public/components/metric_component.tsx#L148
It is a display configuration of the metric (label) to be shown. So, that's why I think, it can't be named in such a way. Possibly, `MetricOptions`?

Reply:
Oh I see, I was mistaken - `MetricOptions` sounds good to me 👍

Reply:
@flash1293, I've replaced `Metric` with `MetricOptions`.